code/script.js

```diff
@@ -1,51 +1,199 @@
   if (sender === 'user') {
+    console.log(message);
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/Usericon.png" alt="User" />  
       </section>
     `
+
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
+    console.log(message);
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/Botdino.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
+  // This makes the chat scroll to the last message
   chat.scrollTop = chat.scrollHeight;
 }
 
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userSay(`I would say ${guess}`);
+    setTimeout(() => whatSize(message), 1500)
+  }
+  else if (questionNumber === 2) {
+    userSay(message)
+    setTimeout(() => ifMonster(message), 1500)
+  }
+  else if (questionNumber === 3) {
+    userSay(message)
+    console.log("right track")
+    setTimeout(() => finish(message), 2000)
+  }
+}
+
+
 // Starts here
 const greetUser = () => {
+  questionNumber = 1
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  botSay("Welcome lifeform! What species do you believe yourself to be?");
 }
+let guess = "";
+button.addEventListener('click', (event) => {
+  event.preventDefault();
+  console.log(event);
+  const userGuess = document.getElementById("name-input").value;
+  console.log(userGuess);
+  guess = userGuess;
+  nextQuestion();
+  nameInput.value = "";
+}, {once: true});
+
+
+const whatSize = () => {
+  questionNumber++
+  botSay(`Let's find out if "${guess}" is true! How smol are you?`);
+  inputWrapper.innerHTML = `
+  <div class=button-wrapper>
+<button id="btnBig">I'm a UNIT</button>
+<button id="btnAverage">Come closer!</button>
+<button id="btnSmall">Tiny, yet brave</button></div>
+  `
+  document
+    .getElementById("btnBig")
+    .addEventListener('click', () => nextQuestion("The biggest"), {once: true});
+  document
+    .getElementById('btnAverage')
+    .addEventListener('click', () => nextQuestion("Let's get comfy"), {once: true});
+  document
+    .getElementById('btnSmall')
+    .addEventListener('click', () => nextQuestion("So smol") , {once: true});
+  }
+
```

Comment:
I learned something new from this usage of `once` value to automatically remove the listened once provoked. Nice work!